scripts/helpers.bats

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bats
+
+load helpers
+
+teardown() {
+	unset DEV_TAGS
+}
+
+@test "assert valid dev_tags set" {
+	export DEV_TAGS='
```

Comment:
Since we are only referencing this variable itself from inside this function, this could be `local` instead of `export`. Then there should be no need for the `teardown` function, as the var will just be scoped to the test function.